eZ/Publish/Core/Repository/Tests/LocationResolver/PermissionAwareLocationResolverTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository\Tests\LocationResolver;
+
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\Repository\LocationResolver\PermissionAwareLocationResolver;
+use eZ\Publish\Core\Repository\Values\Content\Location;
+use PHPUnit\Framework\TestCase;
+
+final class PermissionAwareLocationResolverTest extends TestCase
+{
+    /** @var \eZ\Publish\API\Repository\LocationService */
+    private $locationService;
+
+    /** @var \eZ\Publish\Core\Repository\LocationResolver\LocationResolver */
+    private $locationResolver;
+
+    public function setUp(): void
+    {
+        $this->locationService = $this->createMock(LocationService::class);
+
+        $this->locationResolver = new PermissionAwareLocationResolver($this->locationService);
+    }
+
+    /**
+     * Test for the resolveLocation() method.
+     *
+     * @covers \eZ\Publish\Core\Repository\LocationResolver\PermissionAwareLocationResolver::resolveLocation()
```

Comment:
```suggestion
     * @covers \eZ\Publish\Core\Repository\LocationResolver\PermissionAwareLocationResolver::resolveLocation
```
trivia: coverage annotation should refer to a method, not its invocation (although indeed technically invocations are covered ;) ).